src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -2,57 +2,311 @@
+    synonyms = ("rank_std", "std")
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return np.std(ranks).item()
+
+
+class Variance(RankBasedMetric):
+    """The ranks' variance."""
+
+    value_range = ValueRange(lower=0, lower_inclusive=True, upper=math.inf)
+    synonyms = ("rank_var", "var")
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return np.var(ranks).item()
+
+
+class MedianAbsoluteDeviation(RankBasedMetric):
+    """The ranks' median absolute deviation (MAD)."""
+
+    value_range = ValueRange(lower=0, lower_inclusive=True, upper=math.inf)
+    synonyms = ("rank_mad", "mad")
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return stats.median_absolute_deviation(ranks).item()
+
+
+class Count(RankBasedMetric):
+    """The ranks' count."""
+
+    value_range = ValueRange(lower=0, lower_inclusive=True, upper=math.inf)
+    increasing = True
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return ranks.size
+
+
+class HitsAtK(RankBasedMetric):
+    """The Hits@k."""
+
+    value_range = ValueRange(lower=0, lower_inclusive=True, upper=1, upper_inclusive=True)
+    synonyms = ("H@k", "Hits@k")
+    increasing = True
+
+    def __init__(self, k: int) -> None:
+        super().__init__()
+        self.k = k
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return np.less_equal(ranks, self.k).mean().item()
+
+
+class AdjustedArithmeticMeanRank(RankBasedMetric):
+    """The adjusted arithmetic mean rank (AMR)."""
+
+    value_range = ValueRange(lower=0, lower_inclusive=True, upper=2, upper_inclusive=False)
+    synonyms = ("adjusted_mean_rank", "amr", "aamr")
+    supported_rank_types = (RANK_REALISTIC,)
+    needs_candidates = True
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return (ArithmeticMeanRank.call(ranks) / expected_mean_rank(num_candidates=num_candidates)).item()
+
 
-ARITHMETIC_MEAN_RANK = "arithmetic_mean_rank"  # also known as mean rank (MR)
-GEOMETRIC_MEAN_RANK = "geometric_mean_rank"
-HARMONIC_MEAN_RANK = "harmonic_mean_rank"
-MEDIAN_RANK = "median_rank"
+class AdjustedArithmeticMeanRankIndex(RankBasedMetric):
+    """The adjusted arithmetic mean rank index (AMRI)."""
+
+    value_range = ValueRange(lower=-1, lower_inclusive=True, upper=1, upper_inclusive=True)
+    synonyms = ("adjusted_mean_rank_index", "amri", "aamri")
+    increasing = True
+    supported_rank_types = (RANK_REALISTIC,)
+    needs_candidates = True
+
+    def __call__(self, ranks: np.ndarray, num_candidates: Optional[np.ndarray] = None) -> float:  # noqa: D102
+        return (
+            1.0
```

Comment:
Can you form any "index" with the equation
`(normal - 1) / (expected - 1)`?

Reply:
No, this is a special case which basically works because `E[MR-1] = (max MR-1)/2`